make/ocm.mk

```diff
@@ -23,6 +23,19 @@ define save_cluster_credentials
 	@$(OCM) get /api/clusters_mgmt/v1/clusters/${OCM_CLUSTER_ID}/credentials | jq -r .admin | tee ocm/cluster-credentials.json
 endef
 
+# Get BYOC clusters that are to be retired in next 2 hours
+define get_byoc_clusters_to_be_retired_soon
+	@$(eval CLUSTER_IDS=$(shell $(OCM) get /api/clusters_mgmt/v1/clusters | \
```

Comment:
@jhellar did you tested this only on Mac? I was hitting an issue on Jenkins slaves (Linux) when running the jq command line-separated with backslash line endings: https://github.com/integr8ly/integreatly-operator/pull/419#issue-378944831

Reply:
I've tested that mostly in container we use for our pipelines and hit no issues with this. By the Jenkins slaves did you meant the new container slaves in PSI?

Reply:
Yep. I would maybe also check the PSI Openstack slave, since this is what master installation job uses. If it works, we should be fine.

Reply:
Checked in both openstack [1] and container slaves [2].

[1] https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Playground/job/jhellarByocCleanup/2/console
[2] https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Playground/job/jhellarByocCleanup/1/console